### ISSUE TITLE
Load more icons

### ISSUE THREAD

**Issue by Juli1a** [FortAwesome/Font-Awesome#17155]
When I click the "Load more icons" button - it has to move me to the place where loaded icons begin, not to leave me at the end of just loaded icons.
In this case, I have to scroll up and remember what icons were last ones and again many times. It forces me and any user to spend more time searching for the right icon.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this.

I can confirm, this behavior is strange. Replicated on Chrome 85 / macOS 10.15

I'm going to mention both @talbs and @robmadole

**Comment by robmadole** [comment]
Yes, we know the search interface needs lots of work. We've got this planned for the near future.

**Comment by sensibleworld** [comment]
Thanks for opening this issue! Since it’s been a while and things have changed quite a bit in Font Awesome since this was originally posted, we’re closing it out to help keep our issue tracker focused on current work.

If this is still important for your project, please feel free to re-open or file a fresh issue with updated details. Thanks so much!
